AmplifyPlugins/API/AWSAPICategoryPluginFunctionalTests/GraphQLModelBased/GraphQLConnectionScenario3Tests+Helpers.swift

```diff
@@ -0,0 +1,104 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+import AWSMobileClient
+@testable import AWSAPICategoryPlugin
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSAPICategoryPluginTestCommon
+
+extension GraphQLConnectionScenario3Tests {
+
+    func createPost(id: String = UUID().uuidString, title: String) -> Post3? {
```

Comment:
Might be better to decouple the creation of the `Post3` with calling mutate.  For example, something like:
`func createPost(id: title) -> Post3`
`func mutatePost(post: Post) -> Post3`

Reply:
that sounds good, `mutatePost` will make more sense since it's passing in some post that was retrieved instead of creating a dummy post (this was also bad because the dummy post created for the update will overwrite some other fields that take default values in the constructor)